db/citations.yml

```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
   volume: '88'
   wwwemail: ''
   year: '2013'
+thatc;arxiv21:
+  author:
+    - Zachary Thatcher
+    - Chia-Hao Liu
+    - Long Yang
+    - Brennan C. McBride
+    - Gia Thinh Tran
+    - Allison Wustrow
+    - Martin A. Karlsen
+    - James R. Neilson
+    - Dorthe B. Ravnsbæk
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: nslsii, xpd, desy, P02.1
+  grant: efrc18
+  journal: arXiv:2112.13300 [cond-mat]
+  month: dec
+  nb: ''
+  note: ''
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: '2112.13300'
```

Comment:
need a `tags` item for building reading lists.   You can come up with anything, but ook at Tim's nmf paper and the PDFitc paper for examples.

Also, we need a `synopsis` item and a `professional_summary` and `public_summary` (new features but you should be able to find a couple of examples to go from)

Reply:
Let me know what you think of my `synopsis`, `professional_summary` and `public_summary`
  
I wasn't able to find an example of the public summary, but did my best to create what I would describe as a public summary.